model.py

```diff
@@ -4990,6 +4990,7 @@ class CachedFeed(Base):
     GROUPS_TYPE = 'groups'
     PAGE_TYPE = 'page'
     RECOMMENDATIONS_TYPE = 'recommendations'
+    SERIES_TYPE = 'series'
```

Comment:
If you're going to make the recommendation feed a grouped feed, I don't think you need a separate SERIES_TYPE. The entire feed--recommendations, same author, and same series--would be cached as RECOMMENDATIONS_TYPE.